e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -579,6 +597,36 @@
               "image": "<Container_Registry>/microsoft/azureiotedge-network-controller:<Build.BuildNumber>-linux-<Architecture>",
               "createOptions": "{\"HostConfig\":{\"Privileged\":\"true\",\"NetworkMode\":\"host\",\"Binds\":[\"/var/run/docker.sock:/var/run/docker.sock\"]},\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}}}"
             }
+          },
+          "metricsCollector": {
```

Comment:
does metrics collector support offline?

Reply:
Metrics collector supports an offline flow through event hub so that it can use edge hub for retries and storage. The flow to log analytics does not support offline.

Reply:
It is running in the host network so this will work for connectivity tests